docs/configuration/v1-guarantees.md

```diff
@@ -23,7 +23,8 @@ The Cortex maintainers commit to ensuring future version of Cortex can read data
 
 Cortex strives to be 100% API compatible with Prometheus (under `/api/prom/*`); any deviation from this is considered a bug, except:
 
-- Requiring the `__name__` label on queries.
+- Requiring the `__name__` label on queries (except for queries to Ingesters).
+- For queries to the `/series` endpoint given time ranges older than the data stored in Ingesters will be ignored.
```

Comment:
Few comments:
- I would name the API `/api/v1/series`
- This also affects `/api/v1/labels` and `/api/v1/label/{name}/values`
- Maybe it's more clear if we say that the API endpoints only query data from ingesters (not long-term storage)

Reply:
I have clarified the API path and added the mentioned endpoints. Also added that data is only retrieved from ingesters, wdyt?

Reply:
LGTM, thanks!